codepipeline/app-deployspec.yml

```
@@ -5,6 +5,11 @@ phases:
   install:
     runtime-versions:
       nodejs: latest
```

```suggestion
      nodejs: 18
```

this is better because we need to use node.js 18.
https://docs.aws.amazon.com/codebuild/latest/userguide/runtime-versions.html

we can use `nodejs: 18.x` if needed to use node.js of specific major and latest minor version

---

[e308bb4](https://github.com/cam-inc/viron/pull/658/commits/e308bb4b101c2b3ef6b70bf690d963b3627409e9)